assets/js/feature-tours/idea-hub-module.js

```diff
@@ -41,20 +41,18 @@ const ideaHubModule = {
 	contexts: [ VIEW_CONTEXT_DASHBOARD, VIEW_CONTEXT_DASHBOARD_VIEW_ONLY ],
 	version: '1.43.0',
 	checkRequirements: async ( registry ) => {
-		const isIdeaHubModuleActive = registry
-			.__experimentalResolveSelect( CORE_MODULES )
-			.isModuleActive( 'idea-hub' );
-		const isIdeaHubModuleConnected = registry
-			.__experimentalResolveSelect( CORE_MODULES )
-			.isModuleConnected( 'idea-hub' );
+		const { __experimentalResolveSelect } = registry;
+		const isIdeaHubModuleConnected = await __experimentalResolveSelect(
```

Comment:
This was missing `await` before so the result wouldn't have been evaluated properly.